apps/condo/lang/de/messages/SHARE_TICKET/email.html.njk

```diff
@@ -0,0 +1,22 @@
+<html style="fmargin: 0; font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 22px; font-weight: 400; line-height: 32px; text-align: left;">
+    <body style="margin: 0; font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 22px; font-weight: 400; line-height: 32px; text-align: left;">
+        <table width="100%" border="0" cellpadding="0" cellspacing="0" style="margin:0; padding:0;font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 16px;">
+            <tr>
+                <td width="10%" align="left"><b style="font-size: 16px;">Unitify</b></td>
+                <td width="80%">&nbsp;</td>
+                <td width="10%" align="right"><b style="font-size: 16px;">CONTROL ROOM</b></td>
+            </tr>
+        </table>
+        <p style="font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 22px; font-weight: 400; line-height: 32px; text-align: left;">Hello!<br />
+        Ticket #{{ message.meta.ticketNumber }} dated {{ date | dateFormat('en') }} has been shared with you.<br />
+        The text of the ticket: "{{ message.meta.details }}"</p>
+        <p>&nbsp;</p>
+        <div><!--[if mso]>
+        <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="{{ env.serverUrl }}/ticket/{{ message.meta.id }}" style="height:40px;v-text-anchor:middle;width:330px;" arcsize="10%" stroke="f" fill="t">
+            <v:fill type="tile" src="" color="#4CD174" />
+            <w:anchorlock/>
+            <center style="color:#ffffff;font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif;font-size:16px;font-weight:bold;">To Unitify ticket's card</center>
+        </v:roundrect>
+        <![endif]--><a href="{{ env.serverUrl }}/ticket/{{ message.meta.id }}" style="background-color:#4CD174;border-radius:4px;color:#ffffff;display:inline-block;font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif;font-size:16px;font-weight:bold;line-height:40px;text-align:center;text-decoration:none;width:330px;-webkit-text-size-adjust:none;mso-hide:all;">To Unitify ticket's card</a></div>
```

Comment:
You should consistently use the Doma brand name throughout all translations. If you require customization, feel free to replace all Doma branding with your own.

Reply:
You can check out this issue #5990